Tests/LegacyUnit/Service/RegistrationManagerTest.php

```diff
@@ -2212,6 +2213,37 @@ public function notifyAttendeeForSendConfirmationFalseNotCallsModifyThankYouEmai
         $this->subject->notifyAttendee($registration, $pi1);
     }
 
+    /**
+     * @test
+     */
+    public function notifyAttendeeForSendConfirmationTrueCallsRegistrationEmailHookMethodsForPlainTextEmail()
+    {
+        \Tx_Oelib_ConfigurationProxy::getInstance('seminars')
+            ->setAsInteger('eMailFormatForAttendees', TestingRegistrationManager::SEND_TEXT_MAIL);
+
+        $registration = $this->createRegistration();
+
+        $hook = $this->createMock(RegistrationEmail::class);
+        $hook->expects(self::once())->method('modifyAttendeeEmail')
+            ->with(self::anything(), self::anything(), 'confirmation');
+        $hook->expects(self::once())->method('modifyAttendeeEmailBody')
+            ->with(self::anything(), $registration, 'confirmation', false);
+        $hook->expects(self::never())->method('modifyOrganizerEmail');
+        $hook->expects(self::never())->method('modifyAdditionalEmail');
+
+        $hookClass = \get_class($hook);
+        $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['seminars'][RegistrationEmail::class][] = $hookClass;
+        GeneralUtility::addInstance($hookClass, $hook);
+
+        $this->subject->setConfigurationValue('sendConfirmation', true);
+        $pi1 = new \Tx_Seminars_FrontEnd_DefaultController();
+        $pi1->init();
+
+        $this->subject->notifyAttendee($registration, $pi1);
+
+        GeneralUtility::makeInstance($hookClass);  // purge added instance in case of failing test
```

Comment:
No end-line comments, please -> please move the comment to the line above the code line.